AutoVOD.sh

```diff
@@ -58,7 +66,7 @@ while true; do
 			FETCHED_GAME=$(echo "$json" | jq -r '.stream_game')
 		fi
 
-		if [ "$STREAMER_TITLE" = null ]; then
+		if [ "$STREAMER_TITLE" = null ] || [ "$STREAMER_TITLE" = "initial_title" ]; then
```

Comment:
When `API_CALLS` is set to `TRUE` from config file and streamer is online at the moment, script always returns this:
```
Trying to fetch stream metadata
Stream seems offline, not able to fetch metadata.
```
Checking the script, I discovered that it's trying to find if `$STREAMER_TITLE` is either `null` or `initial_title` while actual title and game name fetched from `twitch-api-wrapper` is stored in `$FETCHED_TITLE` and `$FETCHED_GAME`. This line could be changed like so:
```bash
if [ "$FETCHED_TITLE" = null ] || [ "$FETCHED_TITLE" = "initial_title" ]; then```